doc/rados/operations/health-checks.rst

```diff
@@ -499,13 +499,18 @@ Filestore OSDs can be listed with:
 
    ceph report | jq -c '."osd_metadata" | .[] | select(.osd_objectstore | contains("filestore")) | {id, osd_objectstore}'
 
-If it is not feasible to migrate Filestore OSDs to Bluestore immediately, you
+In order to upgrade to Reef or later releases, any Filestore OSDs must first be
+migrated to BlueStore.
+For releases before Reef, If it is not feasible to migrate Filestore OSDs to BlueStore immediately, you
 can silence this warning temporarily with:
 
 .. prompt:: bash $
 
    ceph health mute OSD_FILESTORE
 
+Since this migration can take considerable time to complete, we recommend that you
+begin the process well in advance of an update to Reef or later releases.
```

Comment:
s/an/any/

I think "any" is a little more on-the nose. Take it or leave it.

Reply:
I think "any" isn't entirely accurate, the advice would then apply *between* Reef minor releases and later to e.g. Reef to S releases.
PTAL at the alternative I'm pushing.